src/Compilers/CSharp/Test/Semantic/Semantics/RecordStructTests.cs

```diff
@@ -3184,7 +3184,11 @@ public static class IsExternalInit
             Assert.Equal("", constructor.GetParameters()[0].GetDocumentationCommentXml());
 
             var property = cMember.GetMembers("I1").Single();
-            Assert.Equal("", property.GetDocumentationCommentXml());
+            AssertEx.Equal(
+@"<member name=""P:C.I1"">
+    <summary>Description for I1</summary>
+</member>
+", property.GetDocumentationCommentXml());
```

Comment:
Could you update the test below (`XmlDoc_Cref`) to check the doc on generated property too?
More cases of references (cref/seealso/...) would be good to cover, if we don't have it already: one parameter/property referencing another parameter/property.